app/templates/components/forms/admin/settings/system/storage-option.hbs

```diff
@@ -0,0 +1,90 @@
+<h3 class="ui header">
+  {{t 'Storage'}}
+  <div class="sub header">
+    {{t 'Where to store data ?'}}
+  </div>
+</h3>
+<div class="field">
+  {{ui-radio label=(t 'Local') current ='local' name='storage' value='local' onChange=(action (mut selectedStorage))}}
+  <span class="small text">
+    ({{t 'stored in'}} /static/media/folder)
+  </span>
+</div>
+<div class="field">
+  {{ui-radio label=(t 'Google Storage') name='storage' value='google' onChange=(action  (mut selectedStorage))}}
+  <span class="small text">
+    ({{t 'See'}}
+    <a href="https://cloud.google.com/storage/docs/migrating#defaultproj" target="_blank" rel="noopener nofollow">
+      {{t 'here'}}
+    </a>
+    {{t 'on how to obtain these keys.'}})
+  </span>
+</div>
+<div class="{{if (not-eq selectedStorage 'google') 'system hidden'}}">
```

Comment:
Don't use a hidden class. Use a full if statement and don't render the things not required.